apps/serialization/gateway.proto

```diff
@@ -291,3 +297,95 @@ syntax = "proto3";
     User user = 1;
     Unit unit = 2;
   }
+
+// BATTLE REPLAY
+
+  enum SkillActionType {
+    ANIMATION_START = 0;
+    EFFECT_TRIGGER = 1;
+    EFFECT_HIT = 2;
+    EFFECT_MISS = 3;
+  }
+
+  enum Stat {
+    health = 0;
+    energy = 1;
+    damage = 2;
+    defense = 3;
```

Comment:
Is there any reason to have these in lower case and the values of SkillActionType in uppercase?